versioned_docs/version-5/migration.mdx

```diff
@@ -695,6 +695,145 @@ actor.subscribe((state) => {
 
 ## Actors
 
+### Use actor logic creators for `invoke.src` instead of functions
+
+:::breakingchange
+
+Breaking change
+
+:::
+
+The available actor logic creators are:
+
+- `createMachine`
+- `fromPromise`
+- `fromObservable`
+- `fromEventObservable`
+- `fromTransition`
+- `fromCallback`
+
+See [Actors](actors.mdx) for more information.
+
+<Tabs>
+<TabItem value="v5" label="XState v5">
+
+```ts
+// ✅
+import { fromPromise, createMachine } from 'xstate';
+
+const machine = createMachine({
+  invoke: {
+    src: fromPromise(async ({ input }) => {
+      const data = await getData(input.userId);
+      // ...
+      return data;
+    }),
+    input: ({ context, event }) => ({
+      userId: context.userId,
+    }),
+  },
+});
+```
+
+</TabItem>
+
+<TabItem value="v4" label="XState v4">
+
+```ts
+// ❌ DEPRECATED
+import { createMachine } from 'xstate';
+
+const machine = createMachine({
+  invoke: {
+    src: (context) => async () => {
```

Comment:
```suggestion
    // ❌ DEPRECATED, use the actor logic creator `fromPromise` in v5.
    src: (context) => async () => {
```

Reply:
@mellson @kevinmaes I appreciate the extra clarity. Maybe this would be sensible for the other examples too?

Reply:
Sure! Should we do a whole pass in another PR then?

Reply:
You can modify this PR